app/models/translation.rb

```diff
@@ -76,6 +81,20 @@ def manifest_translated_phrases
     @manifest_translated_phrases ||= download_translated_phrases("name_description.xml")
   end
 
+  def push_published_to_s3
+    if resource.uses_onesky?
+      ActiveRecord::Base.transaction do
+        phrases = manifest_translated_phrases
+        name_desc_onesky(phrases)
+        create_translated_attributes(phrases)
+      end
+    end
+
+    Package.new(self).push_to_s3
+  rescue => e
+    Rollbar.error(e)
```

Comment:
as far as I know, from the admin tool we will have no way of accessing what this error is or associating it with a specific translation.

Can we store the error string on a new field on the translation object (`publishing-error`) that we will be able to read from the admin tool out of the translation draft model?

Reply:
@frett sorry, I had changed this last night but I forgot to commit :facepalm: